src/main.js

```diff
@@ -137,9 +138,13 @@ function getRandomIndex(array) {
 }
 
 function randomizePoster() {
-  updateTitle.innerText = titles[getRandomIndex(titles)];
-  updateQuote.innerText = quotes[getRandomIndex(quotes)];
-  updateImg.src = images[getRandomIndex(images)];
+  var newTitle = titles[getRandomIndex(titles)];
+  var newQuote = quotes[getRandomIndex(quotes)];
+  var newImage= images[getRandomIndex(images)]
+  updateTitle.innerText = newTitle;
+  updateQuote.innerText = newQuote;
+  updateImg.src = newImage;
```

Comment:
These three lines are repeated twice in your code. You can put these three lines in their own function named updateMainPosterHTML that takes a image src, title, and quote as arguments. By doing this you can also move the three global variables updateTitle, updateQuote, and updateImg into the new updateMainPosterHTML.